webui/react/src/components/kit/SelectFilter.tsx

```diff
@@ -0,0 +1,152 @@
+import { Select } from 'antd';
+import type { DefaultOptionType, LabeledValue, RefSelectProps, SelectValue } from 'antd/es/select';
+import React, { forwardRef, useCallback, useMemo, useState } from 'react';
```

Comment:
@keita-determined we actually do use `ref` except it is only in the `MetricSelectFilter`  you can look at that implementation and see.

Reply:
youre right. i overlooked it

Reply:
We do only use it for the `blur` functionality, and it is the only place we use `onBlur` for the `SelectFilter` I will look and see if we need the `Blur` functionality. If we can get rid of it we can get rid of the extra `OnBlur` prop and the `ref` which would simplify the component.

Reply:
According to https://github.com/determined-ai/determined/pull/6102#discussion_r1123816166, i assume we can remove `ref` from `Select` component since `MetricSelect` was the only place `ref` was used.